CHANGES.rst

```diff
@@ -53,6 +53,8 @@ extract_2d
 
 - Fix slice limits used in extraction of WFSS 2D cutouts. [#7312]
 
+- JP-3007 Add keywords for source RA and Dec for WFSS extractions [#7372]
```

Comment:
We do not put `JP` ticket numbers in the change log since they have no meaning to external users.

Also make sure it is clear that these are added to WFSS mode. BTW, by adding them at this location they will be added to TSGRIM observations as well. If that's not the intention, the PR needs to be modified but I assume this is what is desired.

Reply:
Agree that PR number is not needed in the change log entry.

Reply:
> BTW, by adding them at this location they will be added to TSGRIM observations as well. If that's not the intention, the PR needs to be modified but I assume this is what is desired.

No, I don't think they'll get populated for TSO grism data. That's handled in the `extract_tso_object` function, while this is in the `extract_grism_objects` function. Granted, at first glance one might think that TSO grism objects could be included in that function, but that's just because the function name is so generic/vague. It really is dedicated to WFSS grism objects.